Library/Formula/bbcolors.rb

```diff
@@ -1,11 +1,14 @@
-require 'formula'
-
 class Bbcolors < Formula
-  homepage 'http://daringfireball.net/projects/bbcolors/'
-  url 'http://daringfireball.net/projects/downloads/bbcolors_1.0.1.zip'
-  sha1 'ce47e5ffbcafb01c21acdf242372f351215a80bf'
+  homepage "http://daringfireball.net/projects/bbcolors/"
+  url "http://daringfireball.net/projects/downloads/bbcolors_1.0.1.zip"
+  sha256 "6ea07b365af1eb5f7fb9e68e4648eec24a1ee32157eb8c4a51370597308ba085"
 
   def install
     bin.install "bbcolors"
   end
+
+  test do
+    mkdir_p "#{ENV["HOME"]}/Library/Application Support/BBColors"
```

Comment:
I know it's the same place. But `testpath/"Library..."` looks less scary than `ENV["HOME"]`.